src/main/java/org/opensearch/knn/index/memory/NativeMemoryEntryContext.java

```diff
@@ -192,6 +269,11 @@ public Integer calculateSizeInKB() {
             return size;
         }
 
+        @Override
+        public void openGraphFile() {
```

Comment:
We should throw UnsupportedOperation exception from this function as we are not implementing this.